docs/modules/documentation/containers/alert/alert-docs.component.html

```diff
@@ -1,56 +1,72 @@
 <header>Alert</header>
-<description>Alerts are used to display short messages that require the user's attention.</description>
+<description>Alerts are used to display short messages that require user attention.</description>
 <import module="AlertModule" path="fundamental-ngx"></import>
 <separator></separator>
 
 <h1>&lt;fd-alert&gt;</h1>
 
 <properties [properties]="{
   inputs: [
-    { name: 'type', description: 'Determines the type of alert - \'error\' (red) or \'warning\' (orange). Defaults to white if no type is provided.' },
-    { name: 'id', description: 'The id of the alert. If omitted, an id will be generated by a HashService.' },
-    { name: 'dismissible', description: 'Shows a dismissible button if set to true. Default is false.' },
-    { name: 'persist', description: 'Boolean, when set to true, the alert will not disappear after a set amount of time.  Default is false.'},
-    { name: 'mousePersist', description: 'Boolean, when set to true, the alert will remain visible as long as the mouse is hovered over it.  The alert will disappear when the mouse leaves the alert.  Default is false.'},
-    { name: 'visibleTime', description: 'Number dictating how long (in milliseconds) the alert will be visible.  Default is 10000.'}
+    { name: 'type', description: 'String. Determines the type of alert. View example below for options. Defaults to white if no type is provided.' },
+    { name: 'id', description: 'String. The id of the alert. If omitted, an id will be generated.' },
+    { name: 'dismissible', description: 'Boolean. Shows a dismiss button. Default is true.' },
+    { name: 'persist', description: 'Boolean. When set to true, the alert will not disappear after a set amount of time.  Default is false.'},
+    { name: 'mousePersist', description: 'Boolean. When set to true, the alert will remain visible as long as the mouse is hovered over it. Default is false.'},
+    { name: 'duration', description: 'Number. Dictates how long in milliseconds the alert will be visible. Default is 10000.'},
```

Comment:
10s seems long for a default. Can we expose this, so its a configurable system wide default?

Reply:
We're gonna talk about it, but it feels slightly outside the scope of the ticket. Maybe if you ad another separate issue?

Reply:
If you don't feel its necessary thats ok with me, I have no requirement for it. I was just thinking if you wanted the default to be 5sec for ex, it would be useful to be able to control the alert timeout from 1 place instead of having to put a new value in every alert invocation in your app.

Reply:
I totally agree that it would be useful, we'll just talk about how we want to go about it etc. So possibly for a future ticket